src/useDCPWorker.tsx

```diff
@@ -619,10 +619,10 @@ const useDCPWorker = ({
         };
 
         sandbox.on('start', onStart);
-        sandbox.on('sliceFinish', onFinish);
+        sandbox.on('sliceEnd', onFinish);
```

Comment:
**improvement:** this MR could be further improved by updating `onFinish`. This event now has measurements that come with it, specifically, the compute time. `onFinish` takes the difference between the slice start and finish events to calculate slice _compute time_. Is `measurements.elapsed` a more accurate calculation? If yes, this value should be used to INCREMENT_COMPUTE_TIME. Confirm that `measurements.elapsed` is in fact a replacement for the calculation done in  `onFinish`. I think it is, but I may be wrong.

Reply:
The sliceEnd event was changed in the spec to no longer emit timing measurements. So, there isn't a way to get those values from the function. Its possible that the `metrics` event could be used to replace the existing calculation done in useDCPWorker, but I'd argue that's a separate MR.

Reply:
Ah I see the spec has changed since I last looked at it. In that case, this change looks good. Its possible that introducing the metrics event would even deprecate the use for this event in the worker-hook. I plan to make a ticket(s) related to how the new worker events should affect this hook. Ill resolve this.